src/com/magento/idea/magento2plugin/magento/files/commands/SaveEntityCommandFile.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+package com.magento.idea.magento2plugin.magento.files.commands;
+
+import com.intellij.lang.Language;
+import com.jetbrains.php.lang.PhpLanguage;
+import com.magento.idea.magento2plugin.actions.generation.generator.util.NamespaceBuilder;
+import com.magento.idea.magento2plugin.magento.files.ModuleFileInterface;
+import com.magento.idea.magento2plugin.magento.packages.Package;
+import org.jetbrains.annotations.NotNull;
+
+public class SaveEntityCommandFile implements ModuleFileInterface {
+    public static final String CLASS_NAME = "SaveCommand";
+    public static final String FILE_EXTENSION = "php";
+    public static final String TEMPLATE = "Magento Save Entity Command Model";
+    private static final String DIRECTORY = "Command";
+    private static final SaveEntityCommandFile INSTANCE = new SaveEntityCommandFile();
+
+    /**
+     * Get singleton object of save command file.
+     *
+     * @return SaveEntityCommandFile
+     */
+    public static SaveEntityCommandFile getInstance() {
```

Comment:
No singletons. And this is not actually  a singleton 🤣

Reply:
Lets use just a constructor.

Reply:
Fixed, thank you!)